packages/legend-query-builder/style/_query-builder-projection.scss

```diff
@@ -65,6 +65,7 @@
     width: 1.6rem;
     min-width: 1.6rem;
     margin-left: 0.5rem;
+    cursor: move;
```

Comment:
You put a note in the 
```
  // NOTE: here we use `move` instead of `grab` since grab requires
  // a more sophisticated control, i.e. when dragging, the cursor should
  // be changed
```
since all our drags use move right now as we no longer sense the cursor upon dragging so it defaults back to pointer, the way I had thought about making cursor grab/grabbing would be to have to add some listener to every place we have drag component when user clicks and hasn't let go and i didn't think that amount of change merited the cursor and that move would still suffice  🤔

Reply:
ok, quite sad but I'll accept it 😢

Reply:
Filed in https://github.com/finos/legend-studio/issues/2073 for future reference 👍